app/helpers/dwp_outcome_helper.rb

```diff
@@ -0,0 +1,16 @@
+module DWPOutcomeHelper
+  def reset_confirm_dwp_status!(legal_aid_application)
+    legal_aid_application.update!(dwp_result_confirmed: nil)
+    legal_aid_application.save!
+  end
+
+  def checking_dwp_status!(legal_aid_application)
+    legal_aid_application.update!(dwp_result_confirmed: false)
+    legal_aid_application.save!
+  end
+
+  def confirm_dwp_status_correct!(legal_aid_application)
+    legal_aid_application.update!(dwp_result_confirmed: true)
+    legal_aid_application.save!
```

Comment:
Is a `save!` required as well as an `update!`? Update[!] is saved.

nit/non-blocker: could have a private method that is shared/called by the other methods in here

```ruby
def dwp_result_confirmation!(val)
  legal_aid_application.update!(dwp_result_confirmed: val)
end
```